corehq/apps/app_manager/models.py

```diff
@@ -2477,7 +2477,18 @@ def wrap(cls, data):
 
     @classmethod
     def new_module(cls, name, lang):
-        detail = Detail(
+        short_detail = Detail(
```

Comment:
I think this is right on the borderline of being too much shared code, I'd probably pull the definitions of `short_detail` and `long_detail` into a shared method.

Reply:
Agreed. The duplication can be avoided here with some effort.
I tried now and was thrown off by the `hasAutocomplete` argument which is not present on the `DetailColumn`.

Reply:
I didn't see that difference before, but I think all of them should be using `hasAutocomplete=True`.

Right now, new basic modules look like this:
![Screen Shot 2021-06-07 at 11 28 51 AM](https://user-images.githubusercontent.com/1486591/121044969-a783ad80-c783-11eb-9387-962d204e2252.png)

New advanced modules look like this, with no dropdown. I think this is an oversight - if you add new properties, they get dropdowns.
![Screen Shot 2021-06-07 at 11 29 07 AM](https://user-images.githubusercontent.com/1486591/121045269-b8342380-c783-11eb-88ec-a47e71fcc527.png)

New shadow modules don't display any properties and the "Add Property" button doesn't work, so you aren't going to make them any worse:
![Screen Shot 2021-06-07 at 11 29 23 AM](https://user-images.githubusercontent.com/1486591/121045893-ddc12d00-c783-11eb-9dfb-f4d28cf146f7.png)